### DISCUSSION TITLE
How uses the {{content}} in my template ?

### DISCUSSION THREAD

**Discussion by rtapia121** [mettle/sendportal#152] [Help]
Hi !! I'am new in developmend in laravel and use this mailing system and I'm reading the documentation but I don't understan  how set the placeholder  { { content }} in  my template ? I try with this:

```
<html>
  <body>
   {{content}}
         more html...
  </body>
</html>
```

I'm not sure if it's the correct way of  use. Thanks !

**Comment by HeadStudios** [comment]
No answers - I am trying to use Sendportal but it's a hit and miss with support - guess that's what you get with a free product :(. Did you ever get this working? Are you still using Sendportal?
